lib/partition_setup.pm

```diff
@@ -212,6 +212,36 @@ sub addlv {
     assert_screen 'expert-partitioner';
 }
 
+sub addboot {
+    my $part_size          = shift;
+    my %default_boot_sizes = (
+        ofw        => 8,
+        uefi       => 256,
+        bios_boot  => 2,
+        zipl       => 500,
+        unenc_boot => 500
+    );
+
+    if (get_var('OFW')) {    # ppc64le always needs PReP boot
+        addpart(role => 'raw', size => (defined($part_size)) ? $part_size : $default_boot_sizes{ofw}, fsid => 'PReP');
```

Comment:
I think this also works: `size => $part_size // $default_boot_sizes{ofw}` - just for info, no need to change :)

Reply:
it should, but I like ternary

Reply:
Me too but in perl we should prefer "$part_size // $default" for falling back to default values still :)